### PR TITLE
Disable marquee selection in UnifiedPeoplePicker

### PR DESCRIPTION
**Pull request by sjadelson**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

MarqueeSelection is not the desired behavior for UPP. This change reverts to EPP behavior - no drag selection. A later change will implement correct behavior.

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx [hunk 1]
```diff
@@ -351,45 +343,43 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
       onCopy={_onCopy}
     >
       <FocusZone direction={FocusZoneDirection.bidirectional} {...focusZoneProps}>
-        <MarqueeSelection selection={selection} isEnabled={true}>
```
Comment (elisabethcvs) [inline packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx:354]: Is there anything else this is doing for us that we'll lose if we remove it?

**Review by elisabethcvs — COMMENTED** [review]
Can you run the accessibility checker to make sure this doesn't break anything? I don't think it will, but should probably double check

File: packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx (hunk 1 above)
Comment (nebhatna) [reply packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx:354]: also let us document this as a comment here so we know

**Review by sjadelson — COMMENTED** [review]
Good suggestion on the acc checker.  No violations with 0, 1, 3, and 10 recipients displayed.

**Comment by layershifter** [comment]
@sjadelson is still a thing? Can you please resolve merge conflicts in this case?

**Comment by sjadelson** [comment]
Thanks for pointing that out.  That change was re-jiggered and submitted for the v8 release.  This one is no longer needed, and has been abandoned.

**Pull request closed without merging**